### PR TITLE
Added RSpec for testing and Removed RubyForge project link

### PR DESCRIPTION
**Pull request by maclover7**
Later on I can add a .travis.yml file to run CI tests, if this gets merged in.

@excid3

### CODE REVIEW

File: spec/support/railtie.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module SimpleCalendar
+  class Rails
+    class Railtie
+      def self.initializer(object)
+      end
+    end
+  end
+end
```
Comment (excid3) [inline spec/support/railtie.rb]: Does this help us test the Railtie inclusion in a Rails app?

**Comment by excid3** [comment]
You guys are awesome. :)

@maclover7 Thanks for this PR. Do you have any recommendations on how to test view helpers and Railties?

**Comment by maclover7** [comment]
Not sure. Maybe check out the tests that Rails uses internally?

**Comment by rodrigoargumedo** [comment]
I'm pretty sure that you might want to take a peek of the Rails code and try adding tests from your code with something like:

```ruby
describe SimpleCalendar do
  it 'renders the week calendar' do
    # Not the best code. At least I try. Hehe.
     expect(render_week).not_to be nil
  end
end

**Pull request closed without merging**